star.cpp

```diff
@@ -990,14 +998,15 @@ bool GuideStar::AutoFind(const usImage& image, int extraEdgeAllowance, int searc
         image.BitsPerPixel, sat_level, image.Pedestal, sat_thresh));
 
     // Before sifting for the best star, collect all the viable candidates
-    double minSNR = pFrame->pGuider->getMinStarSNR();
+    double minSNR = pFrame->pGuider->GetAFMinStarSNR();
+    double maxHFD = pFrame->pGuider->GetMaxStarHFD();
     foundStars.clear();
     for (std::set<Peak>::reverse_iterator it = stars.rbegin(); it != stars.rend(); ++it)
     {
         GuideStar tmp;
-        tmp.Find(&image, searchRegion, it->x, it->y, FIND_CENTROID, pFrame->pGuider->GetMinStarHFD(), pCamera->GetSaturationADU(), FIND_LOGGING_VERBOSE);
+        tmp.Find(&image, searchRegion, it->x, it->y, FIND_CENTROID, pFrame->pGuider->GetMinStarHFD(), pFrame->pGuider->GetMaxStarHFD(), pCamera->GetSaturationADU(), FIND_LOGGING_VERBOSE);
         // We're repeating the find, so we're vulnerable to hot pixels and creation of unwanted duplicates
-        if (tmp.WasFound() && tmp.SNR >= minSNR)
+        if (tmp.WasFound() && tmp.SNR >= minSNR && tmp.HFD <= maxHFD)

```

Comment:
I believe this check is redundant and can be removed since if the HFD > maxHFD then `tmp.WasFound()` will be `false` and the check is unreachable.
```suggestion
        if (tmp.WasFound() && tmp.SNR >= minSNR)
```